ui/src/translations/fr.json

```diff
@@ -136,25 +125,7 @@
   "value": "Valeur",
   "prometheus_not_available": "Prometheus n'est pas disponible.",
   "please_refer_to": "Veuillez vous référer à",
-  "import_solution_hint": "Vous devez importer une solution pour préparer votre environnement",
   "redirecting": "Redirection…",
-  "documentation": "Documentation",
```

Comment:
Navbar tests are failling because I think this entry should not have been removed

Reply:
I just realized it as well! Thanks for checking!